js/player.js

```diff
@@ -10,7 +10,6 @@ class Player {
       return this.hand.shift();
     } else if (this.hand === []) {
       checkEmptyDeck(this.id)
```

Comment:
I would see if instead of calling this main.js function, is there something you could do in game or main so that playCard isn't called when their hand is empty?

Reply:
Will do in a new branch